scripts/jenkins/groovy/kerberosCommands.groovy

```diff
@@ -39,15 +45,26 @@ private GString getCommandHadoop(final stageConfig, final spnegoAuth, final bool
         hadoopClasspath = "export HADOOP_CLASSPATH=\$(cat /opt/hive-jdbc-cp)"
     }
     def securityArgs = ""
-    if (steam) {
+    if (impersonate) {
         securityArgs = "-principal steam/localhost@H2O.AI -keytab /etc/hadoop/conf/steam.keytab -run_as_user jenkins"
     }
+    def tokenPreparation = ""
+    def usePreparedToken = ""
+    if (prepareToken) {
+        tokenPreparation = """export HADOOP_CLASSPATH=\$(cat /opt/hive-jdbc-cp)
+            hadoop jar h2o-hadoop-3/h2o-cdh6.3-assembly/build/libs/h2odriver.jar 
```

Comment:
Should the path to the jar be parametrized the same way as it is on https://github.com/h2oai/h2o-3/pull/4479/files#diff-6e5d07024fa7974b3ccb6899740e102bL48?

Reply:
good catch!